crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -3635,6 +3640,10 @@ fn minify_number(num: f64) -> String {
         original.replace_range(0..1, "");
     }
 
+    if original.starts_with("-0.") {
```

Comment:
terser would parse `-0.5` as Unary ( Number (0.5)), so in here we need to treat negative number too

Reply:
Sounds like a bug somewhere, there are no negative numbers in JS, it should be always `Unary + Number`,

Reply:
@alexander-akait minifier generates negative numbers because it's much easier to match on